portfolio/pom.xml

```diff
@@ -23,8 +23,23 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+     <groupId>com.google.code.gson</groupId>
+     <artifactId>gson</artifactId>
+     <version>2.8.6</version>
+    </dependency>
+
+    <dependency>
+     <groupId>com.google.appengine</groupId>
+     <artifactId>appengine-api-1.0-sdk</artifactId>
+     <version>1.9.59</version>
+    </dependency>
+
   </dependencies>
 
+  
```

Comment:
These newlines are probably unnecessary. Not a big deal either way, just a stylistic thing.